packages/core/src/amazonqGumby/chat/controller/messenger/messenger.ts

```diff
@@ -224,7 +231,64 @@ export class Messenger {
                     message: 'Q Code Transformation',
                     formItems: formItems,
                 },
-                'TransformForm',
+                'LanguageUpgradeTransformForm',
+                tabID,
+                false
+            )
+        )
+    }
+
+    public async sendSQLConversionProjectPrompt(projects: TransformationCandidateProject[], tabID: string) {
+        const projectFormOptions: { value: any; label: string }[] = []
+
+        projects.forEach((candidateProject) => {
+            projectFormOptions.push({
+                value: candidateProject.path,
+                label: candidateProject.name,
+            })
+        })
+
+        const formItems: ChatItemFormItem[] = []
+        formItems.push({
+            id: 'GumbyTransformSQLConversionProjectForm',
+            type: 'select',
+            title: 'Choose a project to transform',
+            mandatory: true,
+            options: projectFormOptions,
+        })
+
+        formItems.push({
+            id: 'GumbyTransformSQLSchemaForm',
+            type: 'select',
+            title: 'Choose the schema of the database',
+            mandatory: true,
+            options: Array.from(transformByQState.getSchemaOptions()).map((schema) => ({
+                value: schema,
+                label: schema,
+            })),
+        })
+
+        this.dispatcher.sendAsyncEventProgress(
+            new AsyncEventProgressMessage(tabID, {
+                inProgress: true,
+                message: 'I can convert your embedded SQL, but I need some more info from you first.',
```

Comment:
nit: `information` instead of `info`?

Reply:
Yes agreed, will have doc writer review all these Strings